cedar-drt/fuzz/Cargo.toml

```diff
@@ -119,4 +119,16 @@ doc = false
 name = "eval-type-directed"
 path = "fuzz_targets/eval-type-directed.rs"
 test = false
-doc = false
\ No newline at end of file
+doc = false
+
+[[bin]]
+name = "human-schema-roundtrip"
+path = "fuzz_targets/human-schema-roundtrip.rs"
+test = false
+doc = false
+
+[[bin]]
+name = "json-then-human-schema-roundtrip"
+path = "fuzz_targets/json-then-human-schema-roundtrip.rs"
+test = false
```

Comment:
Can you change the target names to `schema-roundtrip` and `json-schema-roundtrip`? "human" looks weird to me.

Reply:
Also thought of one more comment: the corresponding target for policies (`roundtrip`) tests both the standard & JSON syntax. Is there a reason the tests should be separate?

Reply:
I personalty prefer multiple targets we can tune independently?